site/app/templates/officeHoursQueue/QueueFilter.twig

```diff
@@ -2,6 +2,23 @@
 {% block popup_id %}filter-settings{% endblock %}
 {% block title %}Queue Settings{% endblock %}
 {% block body %}
+  <b>Help With Regex Patterns</b>
+    <a id="regex-help-icon" onclick="toggleRegexHelp()" onkeypress="toggleRegexHelp()" title="Help with regex patterns" aria-label="help with regex patterns" tabindex="0" style="padding:.5rem;">
+        <i class="fas fa-question"></i>
+    </a>
+    <div class="regex-help" style="display: none;">
+        Regex Patterns are to make sure your students enter the correct information
+        into the "contact information" field. Copy the patterns associated with the type of information you wish
+        your students to provide (copy the pound symbols too).<br>
+        <b>NOTE: REGEX PATTERNS ARE NOT REQUIRED</b><br>
+        <a href="https://www.php.net/manual/en/reference.pcre.pattern.syntax.php" target="_blank">Write Your Own Pattern</a><br>
```

Comment:
Do we want to link to a page in the PHP documentation or would it be better to link a page on submitty.org?

Reply:
I think it'd be better to link the php documentation because it has everything someone would need to figure out how to write their own pattern. It's super detailed and I think it'd be a waste of time to make our own documentation for regex.